cmd/buildah/from.go

```diff
@@ -100,8 +102,17 @@ func fromCmd(c *cli.Context) error {
 		return err
 	}
 
+	transport := ""
+	arr := strings.SplitN(args[0], ":", 2)
+	if len(arr) == 2 {
+		if _, ok := util.Transports[arr[0]]; ok {
+			transport = arr[0]
+		}
```

Comment:
Shold we return an error if this is not ok?

Reply:
No, that means the user omitted a transport and is using the Default Transport.

Reply:
No leaving as an empty string just means it will be the Default Transport, which the code handles later on.

Reply:
Right, but if you set it to DefaultTransport here, the code won't have to handle it later and I think it would make this code clearer.  Your call, it's a nit level.

Reply:
setting transport to DefaultTransport on line 105.